javaagent-tooling/src/main/java/io/opentelemetry/javaagent/tooling/PropagatorsInitializer.java

```diff
@@ -50,15 +70,31 @@ public static void initializePropagators(List<String> propagators) {
       OpenTelemetry.setGlobalPropagators(
           DefaultContextPropagators.builder()
               .addTextMapPropagator(HttpTraceContext.getInstance())
+              .addTextMapPropagator(W3CBaggagePropagator.getInstance())
               .build());
       return;
     }
 
     DefaultContextPropagators.Builder propagatorsBuilder = DefaultContextPropagators.builder();
 
     List<TextMapPropagator> textPropagators = new ArrayList<>(propagators.size());
-    for (String propagatorId : propagators) {
-      TextMapPropagator textPropagator = TEXTMAP_PROPAGATORS.get(propagatorId.trim().toLowerCase());
+    List<String> propagatorIds =
+        propagators.stream()
+            .map(propagator -> propagator.trim().toLowerCase())
+            .collect(Collectors.toList());
+
+    if (propagatorIds.remove(JAEGER)) {
+      // Jaeger handles both tracing and baggage
+      propagatorsBuilder.addTextMapPropagator(JaegerPropagator.getInstance());
+      log.info("Added " + JaegerPropagator.getInstance() + " propagator");
```

Comment:
doesn't seem important enough to log by default (default is info)
```suggestion
      log.debug("Added " + JaegerPropagator.getInstance() + " propagator");
```